$NetBSD: patch-src_Makefile.in.in,v 1.2 2016/04/13 13:52:27 hauke Exp $

Directly substitute the local value of etcdir (``../etc/'')
into src/Makefile.in.in.  This allows for overriding etcdir via
INSTALL_MAKE_FLAGS at install-time so that destdir-style installation
will work.

liboss support

SunOS 5 bmake / gmake fixes

--- src/Makefile.in.in.orig	2015-01-29 15:04:29.000000000 +0000
+++ src/Makefile.in.in
@@ -72,8 +72,8 @@ CPPFLAGS=@CPPFLAGS@
 LDFLAGS=@LDFLAGS@
 
 c_switch_all=@c_switch_all@
-ld_switch_all=@ld_switch_all@
-ld_libs_all=@ld_libs_all@
+ld_switch_all=@ld_switch_all@ @LDFLAGS@
+ld_libs_all=@ld_libs_all@ @LIBOSSAUDIO@
 ld_dynamic_link_flags=@ld_dynamic_link_flags@
 
 extra_objs=@extra_objs@
@@ -163,9 +163,9 @@ ldflags += -Wl,--script=s/cygwin.sc
 
 #ifdef SOLARIS2
 %.o : %.c
-#else
-.c.o:
+	$(CC) -c $(cflags) $<
 #endif
+.c.o:
 	$(CC) -c $(cflags) $<
 
 ## Create preprocessor output (debugging purposes only)
@@ -329,7 +329,7 @@ othersrcs = $(otherobjs:.o=.c)
 LIBES = $(lwlib_libs) $(malloclib) $(ld_libs_all) $(lib_gcc)
 
 #ifdef I18N3
-mo_dir = ${etcdir}
+mo_dir = ../etc/
 mo_file = ${mo_dir}emacs.mo
 #endif
 
@@ -658,7 +658,7 @@ libextcli_Xlib.so.1: ${external_client_x
 #endif /* EXTERNAL_WIDGET */
 
 config.h: ${srcdir}/config.h.in
-Emacs.ad.h: ${srcdir}/${etcdir}Emacs.ad
+Emacs.ad.h: ${srcdir}/../etc/Emacs.ad
 
 config.h sheap-adjust.h paths.h Emacs.ad.h :
 	@echo "The file $@ needs to be re-generated."
